xcode/Subconscious/Shared/Parsers/Subtext.swift

```diff
@@ -643,16 +643,18 @@ extension Subtext {
 extension Subtext {
     /// Derive an excerpt
     func excerpt(fallback: String = "") -> String {
-        for block in blocks {
-            switch block {
-            case .empty:
-                continue
-            default:
-                let block = String(block.body())
-                return block.isEmpty ? fallback : block
+        // Filter out empty blocks
```

Comment:
Mentioned above but we should introduce a `Subtext.truncate(maxBlocks: Int) -> Subtext` method and use that instead of flattening to a string and then splitting again.

Reply:
NVM, per above.